mrp_stock_analytic/models/mrp_production.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 ACSONE SA/NV
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MrpProduction(models.Model):
+    _name = "mrp.production"
+    _inherit = ["mrp.production", "analytic.mixin"]
```

Comment:
This seems redundant, as it is present in Odoo 17 because of this change: https://github.com/odoo/odoo/commit/f13d537a42ff1a2713484d0e7c39b73fbb87a7fb#diff-35844959b2eb04809756ac5ad61ac87e18b9064ee8ad4036b96e908adc598f88R11

Reply:
Seems to have been a leftover from the 16 version, at which point this maybe wasn't there. Changed in https://github.com/OCA/account-analytic/pull/644/commits/f18ed0b377c7cd48375b17a6546aa62174dbe3df.